
Running tests...
----------------------------------------------------------------------
F
======================================================================
FAIL [0.001s]: test_should_issue_hello_world_message (helloworld_tests.HelloWorldTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/pybuilder/src/unittest/python/helloworld_tests.py", line 12, in test_should_issue_hello_world_message
    verify(out).write("hello Aman")
  File "/usr/lib/python2.7/dist-packages/mockito/invocation.py", line 111, in __call__
    verification.verify(self, len(matched_invocations))
  File "/usr/lib/python2.7/dist-packages/mockito/verification.py", line 63, in verify
    raise VerificationError("\nWanted but not invoked: %s" % (invocation))
VerificationError: 
Wanted but not invoked: write('hello Aman')

----------------------------------------------------------------------
Ran 1 test in 0.002s

FAILED (failures=1)

Generating XML reports...
